CreditManagement/models.py

```diff
@@ -438,14 +492,43 @@ class UserCredit(models.Model):
     user = models.OneToOneField(User, primary_key=True,
                                   db_column='id',
                                 on_delete=models.DO_NOTHING)
-    balance = models.DecimalField(blank=True, null=True, db_column='balance', decimal_places=2, max_digits=6)
+    balance = models.DecimalField(blank=True, null=True,
+                                  db_column=AbstractTransaction.balance_annotation_name,
+                                  decimal_places=2,
+                                  max_digits=6)
+    balance_fixed = models.DecimalField(blank=True,
+                                        null=True,
+                                        db_column=FixedTransaction.balance_annotation_name,
+                                        decimal_places=2,
+                                        max_digits=6)
+
+    def negative_since(self):
+        """
+        Compute the date in the confirmed moments that
```

Comment:
that ...?

Reply:
"Compute the date from the balance_fixed table when the users balance has become negative"
new text